src/tasks/task.dto.ts

```diff
@@ -1,5 +1,5 @@
 export interface TaskDto {
   title: string;
-  description: string;
+  description?: string;
```

Comment:
Nope. This makes the property optional, so it can be defined or not. You need to do 
```
  description: string | null;
```